code/style.css

```diff
@@ -1,7 +1,551 @@
+  border-width: 0 3px 3px 0;
+  top: 3px;
+  left: 5px;
+  transform: rotate(45deg);
+
+}
+
+button {
+  background-color: transparent;
+  color: #212121;
+  border: 1px solid black;
+  border-radius: 5px;
+  padding: 7px 17px;
+  margin: auto;
+  margin-bottom: 2rem;
+  margin-top: 1rem;
+  text-align: center;
+  -webkit-apperance: none;
+  -moz-appearance: none;
+  appearance: none;
+}
+
+button:hover {
+  background-color: black;
+  color: #EED2D5;
+  transition: all 1.5s ease;
+}
+
+/* Rolling text */
+
+.rolling-text-bottom {
+  font-family: 'Playfair Display';
+  font-size: 6vw;
+  font-style: normal;
+  font-weight: normal;
+  margin-top: 4rem;
+  animation-name: scroll;
+  animation-duration: 10s;
+  position: relative;
+  animation-iteration-count: infinite;
+  animation-timing-function: linear;
+}
+
+@keyframes scroll {
+  from { left: 100%; }
+  to { left: -80%; }
+}
+
+/* Styling of footer*/
+
+footer {
+  padding: 3rem 0 1rem 0;
+  border-top: 15px solid #212121;
+  margin-top: 1rem;
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+ul {
+  list-style-type: none;
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+  padding: 0;
+  align-items: center;
+}
+
+ul a {
+  color: black;
+}
+
+li:hover {
+  background-color: black;
+  color: #EED2D5;
+  transition: all 1.5s ease;  
+  padding: 4px 4px;
+  border-radius: 5px;
+}
+
+.footer-text {
+  font-size: 12px;
+  font-family: 'poppins';
+  font-weight: normal;
+  text-align: center;
+  margin: 2rem 0 1rem 0;
+  color: black;
+}
+
+/* On screens that are 768px or more (tablet) */
+
+@media screen and (min-width: 768px) {
+  
+  body {
+    border: solid #212121 34px;
+  }
+
+  .hero-title-wrapper {
+    margin-top: 120px;
+  }
+
+  .hamburger-menu {
+    display: none;
+  }
+
+  .desktop-menu {
+    display: flex;
+  }
+
+  .rolling-brand-text {
+    font-size: 1.5rem;
+    margin-top:2rem;
+    font-weight: normal;
+    animation-duration: 10s;
+  }
+
+  h2 {
+    margin: 12rem 10rem;
+    padding: 50px;
+    font-size: 2.5vw;
+  }
+
+  h4 {
+    font-size: 1.5rem;
+    padding-bottom: 3rem;
+  }
+
+  h5 {
+    font-size: 3rem;
+  }
+
+  p {
+    font-size: 1rem;
+  }
+
+  .active + .project-content {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+  }
+
+  .project-title::before {
+    right: 10rem;
+  }
+
+  .project-title {
+    height: 140px;
+    font-size: 20px;
+  }
+
+  .project-title h4 {
+    font-size: 2.5rem;
+  }
+
+  .project-image {
+    width: 50%;
+    margin-right: 1rem;
+  }
+
+  .content-text {
+    margin-top: 1rem;
+  }
+
+  .form-wrapper {
+    width: 50%;
+  }
+
+  button {
+   margin-top: 2rem;
+   margin-bottom: 2rem;
+  }
+
+  footer {
+    margin-top: 3rem;
+    border-top: 34px solid #212121;
+  }
+
+  li{
+    font-size: 1.5rem;
+    margin:0.5rem;
+  }
+
+}
+
+/* On screens that are 992px or more (desktop) */
+
+@media screen and (min-width: 992px) {  
```

Comment:
Correct me if I'm wrong. It looks like this we don't need this part as the tablet and desktop view have the same layout?

Reply:
ohh. no I just forgot to delete it when I realized I didn't want anything specific for desktop :)